lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -1595,6 +1603,7 @@ class TextEditingChannel {
         // There's no API to request autofill on the web. Instead we let the
         // browser show autofill options automatically, if available. We
         // therefore simply ignore this message.
+        command = const TextInputRequestAutofill();
```

Comment:
Maybe remove the `late` on line 1555?

Reply:
according to the comments I think it's safe to just return here.

Reply:
Just keep it consistent with other unsupported commands, such as 'TextInputSetCaretRect'.